File: Back/src/main/java/web/practicas/demo/model/dto/ProyectoAddDTO.java
package web.practicas.demo.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import web.practicas.demo.model.entidades.Atributo;

import java.util.List;
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class ProyectoAddDTO {

    private String nombre;
    private String Equipo;
    List<ProyectoAtributoDTO>atributos;
}

File: Front/src/app/Interfaces/dto/ProyectoAddDTO.ts
import { ProyectoAtributoDTO } from "./ProyectoAtributoDTO";
import { ProyectoDTO } from "./ProyectoDTO";

export class ProyectoAddDTO{
    constructor(nombre:string,Equipo:string,atributos:ProyectoAtributoDTO[]){
        this.nombre=nombre;
        this.equipo=Equipo;
        this.atributos=atributos;
    }
    nombre:string;
    equipo:string;
    atributos:ProyectoAtributoDTO[];

}